datafusion/src/logical_plan/expr.rs

```diff
@@ -1762,4 +1794,76 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn normalize_cols() {
```

Comment:
I added some additional unit test coverage here for normalize when I was writing the `unnormalize` versions